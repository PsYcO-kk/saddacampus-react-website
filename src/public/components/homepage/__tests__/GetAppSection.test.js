import {shallow} from 'enzyme';
import React from 'react';
import AppSection from '../GetAppSection.js';

test("testing for App Section page component" , ()=>{
    const wrapper = shallow(<AppSection />)
    expect(wrapper).toMatchSnapshot();
})
