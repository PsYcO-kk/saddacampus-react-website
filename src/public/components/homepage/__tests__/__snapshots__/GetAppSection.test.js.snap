// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for App Section page component 1`] = `
<section
  className="section-gap-half"
>
  <div
    className="container jumbotron"
    style={
      Object {
        "borderRadius": "0",
      }
    }
  >
    <div
      className="row align-items-center"
    >
      <div
        className="col-lg-4 about-left"
      >
        <img
          alt=""
          className="img-fluid"
          src="/assets/images/application.png"
        />
      </div>
      <div
        className="col-lg-8 about-right"
      >
        <h1>
          The Best Food Delivery App
        </h1>
        <h4
          style={
            Object {
              "fontWeight": "normal",
              "marginTop": "3%",
            }
          }
        >
          Now you can make food happen pretty much wherever you are thanks to the free easy-to-use Food Delivery.
        </h4>
        <div
          style={
            Object {
              "marginTop": "2%",
            }
          }
        >
          <a
            href="https://play.google.com/store/apps/details?id=com.saddacampus.app"
            style={
              Object {
                "cursor": "pointer",
              }
            }
          >
            <img
              alt=""
              className="img-fluid"
              src="/assets/images/google-play-badge.png"
              style={
                Object {
                  "maxWidth": "40%",
                }
              }
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</section>
`;
